pkgs/development/libraries/pango/default.nix

```diff
@@ -75,6 +75,8 @@ stdenv.mkDerivation rec {
     "-Dintrospection=${if withIntrospection then "enabled" else "disabled"}"
   ] ++ lib.optionals (!x11Support) [
     "-Dxft=disabled" # only works with x11
+  ] ++ lib.optionals withDocs [
+    "-Ddatadir=${placeholder "devdoc"}"
```

Comment:
I do not really like changing the whole `datadir`, since it can potentially contain other stuff in the future.